shared-ui/styled-components/Background.styles.ts

```diff
@@ -1,9 +1,15 @@
 import styled from 'styled-components';
 
-const StyledDayBackground = styled.img`
+const StyledBackgrounds = styled.img`
   width: 100%;
   position: absolute;
   z-index: -1;
 `;
 
-export { StyledDayBackground };
+const StyledStar = styled.img`
```

Comment:
The sun and moon should be wayyyy smaller on mobile. Add a breakpoint and set widths on mobile to make it smaller. You might also need to add a width for the desktop sizing if the mobile overrides it in the media query

Reply:
done by Lisa, resolving